faq.html

```diff
@@ -127,11 +151,11 @@ <h3 class="question">What is Real Dev Squad?</h3>
               </p>
             </div>
           </div>
-          <div class="faq">
-            <a class="faq_link" href="#How-do-I-contribute-to-RDS">
-              <div class="faq__title">
+          <div class="faq" type="button">
```

Comment:
If you are using type="button" for div then we might need to give aria-label, keydown event, tabIndex. Can we make use of button instead?

Reply:
The whole earlier would be refactor in another Pr